src/actions/AuthActions.js

```diff
@@ -66,14 +70,16 @@ export const checkAndSetToken = (passedToken) => {
             if (token === tokens.GUEST) {
                 dispatch({
                     type: GUEST_TOKEN_SET,
-                    payload: {token, appReady: true}
+                    payload: {token, appReady: false}
                 });
             } else {
                 dispatch({
                     type: FACEBOOK_LOGIN_SUCCESS,
-                    payload: {token, appReady: true}
+                    payload: {token, appReady: false}
```

Comment:
I'm playing around with having the splash screen and `<AppLoading />` always show until everything is ready. Sending out `appReady: false` after successfully logging in should trigger the `<AppLoading />` screen, but it doesn't seem to be doing it as expected. I'm going to need to investigate it more.